pavelib/assets.py

```diff
@@ -718,6 +705,28 @@ def execute_compile_sass(args):
         )
 
 
+def get_parsed_option(options, name, default=None):
+    """
+    Options to a command are provided as string value. Convert it to list where a list is expected
+    after fetching an option specified by name
+    Args:
+        options: Command line arguments passed via paver command.
+        name: name of option to parse
+        default: if option not in options, return default
+    """
+    option = default
```

Comment:
This is done because we want to return a pre defined value if we can't find any from given `options` parameter. That default value is passed to method as `default`. In proceeding code if we can't find any value demanded we will just return option and it is already set to default.